episodes/filetypes-and-metadata.md

```diff
@@ -177,6 +177,12 @@ Only certain file types will support browsing metadata via the 'OME Tree Widget'
 in napari-aicsimageio. The plugin is still under development - so more formats 
 are likely to be supported in future!
 
+Napari-aicsimageio also has [quite strict limits
```

Comment:
What's the intention of the wording "quite strict"?. It implies we have an opinion on the appropriateness of the limits. Would it be better to just say "has limits on the size of the images"?

Reply:
Happy to change this to just 'has limits'. I put 'quite strict' as many other pieces of image analysis software allow you to open much larger images directly into memory e.g. if you have a workstation with large amounts of RAM

Reply:
Maybe we could then say, Napari-aicsimageio has size limits that tend to be lower than other programs (i.e. QuPath, Fiji) in this area.

Reply:
I've changed it to just 'has limits' now. I was wary of comparing directly to other programs as some of them have quite complex size requirements e.g. QuPath relies a lot on chunked, pyramidal file formats rather than loading everything into memory so it becomes difficult to compare.

I'll go ahead and merge this later today - so let me know if anything else needs changing!